js/index.js

```diff
@@ -1,13 +0,0 @@
-/*Event listener for changing nav button style based on state */
-
-const menuItem = document.querySelectorAll(".menuItem").forEach(item => {
-    item.addEventListener("click", () => {
-        if (!item.classList.contains("active")) {
-            item.classList.add("active");
-        };
-        elseif (item.classList.contains("active")); {
```

Comment:
FYI @kydkennedy the `}; elseif` is a syntax error. `} else if` would be correct.

Though, I ended up removing this because syncing the active menu list item with user scroll is actually kind of difficult. Can totally add back in if it's a must-have though!

Reply:
Thanks Josh!
It's not a must-have, more like a nice to have if we have time.